cases/failed-csi-provisioning-aws/stateful-set.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: nginx
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  serviceName: "nginx"
+  replicas: 3 
+  minReadySeconds: 10 
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: nginx
+        image: registry.k8s.io/nginx-slim:0.24
+        ports:
+        - containerPort: 80
+          name: nginx
+        volumeMounts:
+        - name: www
+          mountPath: /usr/share/nginx/html
```

Comment:
we can try this but i wouldn't be surprise if this crashes nginx because the folder will be empty